viewer/packages/360-images/src/icons/IconCollection.ts

```diff
@@ -85,6 +125,29 @@ export class IconCollection {
     };
   }
 
+  private computeProximityPoints(octree: IconOctree, iconSprites: InstancedIconSprite): BeforeSceneRenderedDelegate {
+    return ({ camera }) => {
+      const points =
+        this._proximityRadius < 1
+          ? this._icons
+          : octree.findPoints(camera.position, this._proximityRadius).map(pointContainer => {
+              return pointContainer.data;
+            });
+
+      let closestPoints = points.sort((a, b) => {
+        return b.position.distanceTo(camera.position) - a.position.distanceTo(camera.position);
+      });
+
+      if (this._proximityPointLimit > 0) {
+        closestPoints = closestPoints.slice(-(this._proximityPointLimit + 1)); //Add 1 to "skipSelf"..
```

Comment:
This syntax confuses me a little. Is this taking the last this._proximityPointLimit points in the array? If so, would it not be cleaner to reverse the sorting order and then do `closestPoints.slice(this._proximityPointLimit)` (this might be missing a +1).

Reply:
Done! 😎 Note that I now do reverse() the list before I set the icons further down in the code. As we talked about, this is to put the transparent icon in the correct order in the scene.